controllers/rcj_soccer_referee_supervisor/referee/utils.py

```diff
@@ -73,3 +76,15 @@ def is_outside(x: int, z: int) -> bool:
         return not (is_in_blue_goal(x, z) or is_in_yellow_goal(x, z))
 
     return False
+
+
+def get_random_item(items: Iterable[Any]) -> Any:
```

Comment:
@Adman Was this really necessary? :slightly_smiling_face:

Reply:
Removed.